server/router.ts

```diff
@@ -3,8 +3,7 @@ import * as sitterController from "./controllers/sitterControl";
 
 const router = express.Router();
 
-router.get("/sitters", sitterController.addSitter);
```

Comment:
http requests were inverted here, get calling addSitter, and post calling getSitter, i have updated it so they call the proper functions.